src/main/java/com/microsoft/sqlserver/jdbc/SQLServerColumnEncryptionCertificateStoreProvider.java

```diff
@@ -25,6 +40,10 @@
     static final String currentUserDirectory = "CurrentUser";
     static final String myCertificateStore = "My";
 
+    // FOR NON WINDOWS MACHINES
+    String keyStorePath = null;
+    char[] keyStorePwd = null;
+
     static {
         if (System.getProperty("os.name").toLowerCase(Locale.ENGLISH).startsWith("windows")) {

```

Comment:
**Reporter**: CHECKSTYLE
**Rule**: com.puppycrawl.tools.checkstyle.checks.sizes.LineLengthCheck
**Severity**: ERROR
**File**: src/main/java/com/microsoft/sqlserver/jdbc/SQLServerColumnEncryptionCertificateStoreProvider.java L48

Line is longer than 80 characters (found 94).

*<this is a auto generated comment from violation-comments-lib F7F8ASD8123FSDF>* *<-731439326>*